scenarioo-client/app/manage/label-colors/label-colors.component.ts

```diff
@@ -0,0 +1,90 @@
+import {Component, OnInit} from '@angular/core';
+import {LabelConfigurationsListResource} from '../../shared/services/labelConfigurationsListResource.service';
+import {LabelConfigurationsResource} from '../../shared/services/labelConfigurationsResource.service';
+import {AvailableColor} from './available-color';
+import {LabelConfiguration} from './label-configuration';
+
+@Component({
+    selector: 'sc-label-colors',
+    template: require('./label-colors.component.html'),
+    styles: [require('./label-colors.component.css').toString()],
+})
+export class LabelColorsComponent implements OnInit {
+    availableColors: AvailableColor[] = [
+        new AvailableColor('#e11d21', '#FFFFFF'),
+        new AvailableColor('#eb6420', '#FFFFFF'),
+        new AvailableColor('#fbca04', '#000000'),
+        new AvailableColor('#009800', '#FFFFFF'),
+        new AvailableColor('#006b75', '#FFFFFF'),
+        new AvailableColor('#207de5', '#FFFFFF'),
+        new AvailableColor('#0052cc', '#FFFFFF'),
+        new AvailableColor('#5319e7', '#FFFFFF'),
+    ];
+
+    labelConfigurations: LabelConfiguration[] = [];
+    successfullyUpdatedLabelConfigurations = false;
+
+    constructor(private labelConfigurationsListResource: LabelConfigurationsListResource,
+                private labelConfigurationsResource: LabelConfigurationsResource) {
+    }
+
+    ngOnInit(): void {
+        this.loadLabelConfigurations();
+    }
+
+    onDelete(configurationToDelete) {
+        this.labelConfigurations = this.labelConfigurations
+            .filter((configuration) => configuration.name !== configurationToDelete.name);
+    }
+
+    onLabelNameChanged() {
+        const lastLabelInList = this.labelConfigurations[this.labelConfigurations.length - 1];
+        if (!lastLabelInList.isEmpty()) {
+            this.labelConfigurations.push(LabelConfiguration.empty());
```

Comment:
why do you use different styles? On line 70 you use the spread operator

Reply:
Yep, that didn't catch my attention. I will use the spread operator here as well.